project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Newsletter button background contrast is not correct
<img width="457" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/0ebe6f77-9411-4a20-af13-104422b2cfb2">